src/components/CartItem.jsx

```diff
@@ -58,6 +59,7 @@ export default class CartItem extends Component {
         >
           -
         </button>
+        <CartQuantity quantity={ this.state } />
```

Comment:
```suggestion
        <CartQuantity quantity={ this.state } />
```
Podemos tirar essa prop, já que não estamos mais utilizando